content/zh-cn/docs/tasks/debug/debug-cluster/windows.md

```diff
@@ -31,7 +31,7 @@ content_type: concept
 1. 我的 Pod 都卡在 “Container Creating” 或者不断重启
 
    确保你的 pause 镜像跟你的 Windows 版本兼容。
-   查看 [Pause 容器](/zh-cn/docs/setup/production-environment/windows/intro-windows-in-kubernetes#pause-container)
+   查看 [Pause 容器](/zh-cn/docs/concepts/windows/intro/#pause-container)
    以了解最新的或建议的 pause 镜像，或者了解更多信息。
```

Comment:
The commented original text should be updated too.

Reply:
Thank you very much for your reminding. I will revise it

Reply:
@Sea-n cc